app/src/main/java/com/example/pocketcloset/adapters/RandomizedAdapter.java

```diff
@@ -0,0 +1,85 @@
+package com.example.pocketcloset.adapters;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.example.pocketcloset.R;
+import com.example.pocketcloset.models.Clothing;
+import com.parse.ParseFile;
+
+import java.util.List;
+
+public class RandomizedAdapter extends RecyclerView.Adapter<RandomizedAdapter.ViewHolder> {
```

Comment:
I would have thought adding this abstraction would have enabled you to delete code elsewhere?